test/audit/AuditingServiceSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 HM Revenue & Customs
+ * Copyright 2019 HM Revenue & Customs
```

Comment:
Failing test in this spec - same reason as with the other backends where the audit detail has changed.